frontend/talentsearch/src/js/components/search/SearchForm.tsx

```diff
@@ -40,6 +41,7 @@ export type FormValues = Pick<
 > & {
   languageAbility: LanguageAbility | typeof NullSelection;
   classifications: string[] | undefined;
+  skills: string[] | undefined;
   cmoAssets: string[] | undefined; // TODO REMOVE WHEN REPLACING CMOASSETS
```

Comment:
ah, there are some remnants of CMOassets to clean up
I think I marked some with todo's but might need to ctrl+F to be safe